hadoop-ozone/dist/pom.xml

```diff
@@ -340,6 +340,7 @@
                   <appendAssemblyId>false</appendAssemblyId>
                   <attach>false</attach>
                   <finalName>ozone-${project.version}-src</finalName>
+                  <tarLongFileMode>gnu</tarLongFileMode>
```

Comment:
Took a quick look, perhaps this should be posix? Ref: [assembly plugin should use posix, not gnu for tarLongFileMode](https://issues.maven.apache.narkive.com/8fUQgyGd/jira-created-mpom-132-assembly-plugin-should-use-posix-not-gnu-for-tarlongfilemode)

Reply:
Strange, link works for me but yes that is the issue.